boba-chain-ops/deployer/deployer.go

```diff
@@ -32,7 +32,7 @@ type Constructor struct {
 
 type Deployment struct {
 	Name     string
-	Bytecode hexutility.Bytes
```

Comment:
I know we discussed this in other channels but so strange.  Really wonder what is up here, but not worth spending the cycles to fix given the simple workaround.  Just wanted to note that this was discussed elsewhere for posterity.